content/docs/command-reference/remove.md

```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
 usage: dvc remove [-h] [-q | -v] [-o | -p] [-f] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to remove.
+  targets        `.dvc` files to remove.
```

Comment:
@efiop we need to do this same for the core repo when possible please. See #3960

Reply:
@jorgeorpinel Wait, that looks wrong. It should be "stages or `.dvc` files to remove". It works with `dvc.yaml` too.

Reply:
I was assuming remove only works with .dvc files like move. OK, great to know, so let's go with this:

```suggestion
  targets        stages (found in `dvc.yaml`) or `.dvc` files to remove.
```

Cc @utkarshsingh99

Reply:
I was thinking of not to mention this yet, only after we release docs for `run`.

Reply:
`remove` only has `--outs` flag now, and `0.94`'s `remove` and `1.0`'s `remove` behavior are completely different.

Reply:
OK, good to know @skshetry . Please open a separate issue or PR from the core repo PR where `remove` is changed (usual process). This PR is about updating "DVC-file" terminology. Thanks